frontend/src/components/SignUpForm.js

```diff
@@ -0,0 +1,78 @@
+import React, { useState } from 'react';
+import { useSelector } from 'react-redux';
+
+import { Profile } from './Profile.js';
+
+import { Heading, Text, UserForm, FormLabel, FormInput, Button } from 'styling/styling';
+
+
+const SIGNUP_URL = 'https://agnes-emma-login.herokuapp.com/users';
+
+export const SignUpForm = () => {
+    const accessToken = useSelector((store) => store.user.login.accessToken);
+
+    const [name, setName] = useState('');
+    const [password, setPassword] = useState('');
+
+    const handleLoginSuccess = (loginResponse) => {
+    };
+
+    const handleLoginFailed = (loginError) => {
+    };
+    
```

Comment:
Seems like maybe you haven't had the time to finish yet? I cannot see an error message in the frontend if I tried to register and it didn't work

Reply:
And you could reset your local states to empty the input fields after pressing the button :)